### ISSUE TITLE
Move graph settings to a dedicated class

### ISSUE THREAD

**Issue by ruslangaripov** [Serg-Norseman/Arbor#30]
To make the `ARBOR graph` class declaration more clear move all `static constexpr` constants to new base class.

I'm talking about this:

``` cpp
{
    . . .
    static constexpr float m_stiffness = 250.0f;
#if defined(__ICL)
    static constexpr float m_repulsion = 10000.0f;
#else
    static constexpr float m_repulsion = 10'000.0f;
#endif
    static constexpr float m_friction = 0.1f;
    static constexpr float m_animationStep = 0.04f;
    static constexpr float m_timeSlice = 0.01f;
    static constexpr float m_energyThreshold = 0.7f;
    static constexpr float m_theta = 0.4f;
    static constexpr bool m_gravity = false;
    static constexpr bool m_autoStop = false;
    . . .
}
```

**Comment by ruslangaripov** [comment]
Such changes cause changes in data members alignment. This definitely increases size of objects. Therefore this ticket is closed as a bad one.

:smiling_imp:

**Comment by ruslangaripov** [comment]
It looks like that (data alignment messing up) is MSFT VS 2015 Update 1 feature. Intel C++ doesn't do that. And it looks like [MSFT VS 2015 Update 2 doesn't](https://blogs.msdn.microsoft.com/vcblog/2016/03/30/optimizing-the-layout-of-empty-base-classes-in-vs2015-update-2-3/)!

Therefore I reopen this ticket and wait for MSFT 2015 Update 2 RTM to be released. :hourglass:

**Comment by ruslangaripov** [comment]
Fixed in 9455f80
